shieldapi/keycloak_utils.py

```diff
@@ -40,14 +36,16 @@ def get_keycloak_openid(
         KeycloakOpenID: An object for communicating with Keycloak via OpenID Connect.
 
     Raises:
-        KeyError: If any required environment variables are missing.
+        KeyError: If any required values are missing.
     """
 
     return KeycloakOpenID(
-        server_url=_get_value(server_url, "KEYCLOAK_HOST"),
+        server_url=_get_value(
+            server_url, "KEYCLOAK_HOST", "http://keycloak:8080/auth/"
+        ),
         realm_name=_get_value(realm_name, "KEYCLOAK_REALM_NAME"),
-        client_id=_get_value(client_id, "KEYCLOAK_CLIENT_ID"),
-        client_secret_key=_get_value(client_secret_key, "KEYCLOAK_CLIENT_SECRET_KEY"),
+        client_id=_get_value(client_id, "KEYCLOAK_CLIENT_ID", "shieldapi"),
+        client_secret_key=_get_value(client_secret, "KEYCLOAK_CLIENT_SECRET"),
```

Comment:
Why do you go back to `client_secret` when the original kwarg from Python is `client_secret_key` ? I think it makes sense for a user to keep the name of the kwarg as close as it is from the original function.

Reply:
The `key` is added by the python keycloak library, Keycloak itself also uses [`client_secret`](https://www.keycloak.org/docs-api/18.0/rest-api/index.html#_getclientsecret)

Reply:
Okay, that's odd. Nervermind